bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```
@@ -379,7 +379,7 @@ spec:
                 - --zap-log-level=info
                 - --zap-time-encoding=rfc3339nano
                 - --feature-gates=+operator.autoinstrumentation.go,+operator.autoinstrumentation.nginx
-                image: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:0.89.0
+                image: ghcr.io/francesl/opentelemetry-operator/opentelemetry-operator:0.89.0-27-g0ad467d
```

```suggestion
                image: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:0.89.0
```

I think we have to revert this.